src/compiler/config/outputs/validate-dist.ts

```diff
@@ -27,8 +27,11 @@ import { validateCopy } from '../validate-copy';
  */
 export const validateDist = (config: d.ValidatedConfig, userOutputs: d.OutputTarget[]): d.OutputTarget[] => {
   const distOutputTargets = userOutputs.filter(isOutputTargetDist);
-  return distOutputTargets.reduce((outputs: d.OutputTarget[], o: d.OutputTargetDist) => {
-    const distOutputTarget = validateOutputTargetDist(config, o);
+
+  const outputs: d.OutputTarget[] = [];
+
+  for (const outputTarget of distOutputTargets) {
+    const distOutputTarget = validateOutputTargetDist(config, outputTarget);
```

Comment:
while I was in here I thought hey, this `.reduce` call is kind of weird, let's just `for ... of` instead